src/tap-windows6.vcxproj.in

```diff
@@ -121,6 +121,10 @@
     <Link>
       <AdditionalDependencies>%(AdditionalDependencies);$(KernelBufferOverflowLib);$(DDK_LIB_PATH)ntoskrnl.lib;$(DDK_LIB_PATH)hal.lib;$(DDK_LIB_PATH)wmilib.lib;ndis.lib;ntstrsafe.lib;wdmsec.lib</AdditionalDependencies>
     </Link>
+    <Inf>
+      <TimeStamp>@PRODUCT_TAP_WIN_MAJOR@.00.00.@PRODUCT_TAP_WIN_MINOR@</TimeStamp>
+      <DateStamp>@PRODUCT_TAP_WIN_RELDATE@</DateStamp>

```

Comment:
That format was before my version number patch (PR #48) -- this is internally merged in @mattock's test executable. So could you please change the TimeStamp definition to 
@PRODUCT_TAP_WIN_MAJOR@.@PRODUCT_TAP_WIN_MINOR@.@PRODUCT_TAP_WIN_REVISION@.@PRODUCT_TAP_WIN_BUILD@

Reply:
Done.